package/MDAnalysis/core/groups.py

```diff
@@ -734,6 +736,11 @@ def center(self, weights, pbc=None, compound='group'):
 
         comp = compound.lower()
         if comp == 'group':
+            if unwrap and pbc:
```

Comment:
This checking for if unwrap and wrap aren't both set is going to be very common, so it makes sense to check this in a decorator, eg how we use `@warn_if_not_unique` or `@_only_same_level`.

This decorator can also handle the deprecation of `pbc` into `unwrap` too....

```python
In [1]: def AorB(func): 
   ...:     def inner(*args, **kwargs): 
   ...:         if kwargs.get('A', False) and kwargs.get('B', False): 
   ...:             raise ValueError("both A and B set to true") 
   ...:         return func(*args, **kwargs) 
   ...:     return inner 
   ...:                                                                                                                                       

In [13]: @AorB 
    ...: def thingy(A=False, B=False): 
    ...:     return bool(A) or bool(B) 
    ...:                                                                                                                                      

In [14]: thingy()                                                                                                                             
Out[14]: False

In [15]: thingy(A=True)                                                                                                                       
Out[15]: True

In [16]: thingy(B=True)                                                                                                                       
Out[16]: True

In [17]: thingy(A=True, B=True)                                                                                                               
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-17-8707bab20e96> in <module>
----> 1 thingy(A=True, B=True)

<ipython-input-1-baa4bbe6971a> in inner(*args, **kwargs)
      2     def inner(*args, **kwargs):
      3         if kwargs.get('A', False) and kwargs.get('B', False):
----> 4             raise ValueError("both A and B set to true")
      5         return func(*args, **kwargs)
      6     return inner

ValueError: both A and B set to true

```